docs/quickstart.ipynb

```diff
@@ -79,24 +90,23 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Crawling through <Catalog id=sentinel-stac>\n",
-      "Crawling through <Collection id=sentinel-2-l1c>\n",
-      "Crawling through <Catalog id=9>\n",
-      "Crawling through <Catalog id=V>\n"
+      "1.0.0\n"
      ]
     }
    ],
    "source": [
-    "while len(cat.get_item_links()) == 0:\n",
-    "    print('Crawling through {}'.format(cat))\n",
-    "    cat = next(cat.get_children())"
+    "print(get_stac_version())"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We can print some information about the catalog, including how many children it has:"
+    "## Crawling Child Catalogs/Collections\n",
+    "\n",
+    "[STAC Collections](https://github.com/radiantearth/stac-spec/tree/master/collection-spec) are used to group related Items and provide aggregate or summary metadata for those Items.\n",
+    "\n",
+    "STAC Catalogs may have many nested layers of Catalogs or Collections within the top-level collection. Our example catalog has one Collection within the main Catalog at ([landsat-8-l1/collection.json](./example-catalog/landsat-8-l1/collection.json). We can list the Collections in a given Catalog using the [`Catalog.get_collections`](https://pystac.readthedocs.io/en/latest/api.html#pystac.Catalog.get_collections) method. This method returns an iterable of PySTAC [`Collection`](https://pystac.readthedocs.io/en/latest/api.html#collection) instances, which we will turn into a `list`."
```

Comment:
2 things:
- There's a stray open paren in the first landsat link that throws off rendering.
- The nbsphinx can't handle backticks in the brackets for links (seen when rendering the quickstart through `make livehtml` in the docs folder). You should remove backticks, e.g. [\`Catalog.get_collections\`] ->[Catalog.get_collections]